packages/client/hmi-client/src/temp/Vegalite.vue

```diff
@@ -37,12 +36,15 @@ for (let j = 0; j < numPoints; j++) {
 
 for (let i = 0; i < numSamples; i++) {
 	let error = 0;
+	let error2 = 0;
```

Comment:
Could you add a comment to explain the purpose of this `error2`?

Reply:
First rule of `/temp` folder: you do not talk about `/temp` folder 😄